### ISSUE TITLE
Disable text on the card

### ISSUE THREAD

**Issue by romquenin** [vasqued2/ha-teamtracker-card#121]
Hello, thank you very much for this card.

I would like to know if it's possible to disable the text "Tele" located in the bottom right corner of the card

![Capture d’écran 2024-09-19 025953](https://github.com/user-attachments/assets/985172d7-9019-4de5-a78b-73609a99cc14)

**Comment by vasqued2** [comment]
Absolutely.  You can use card-mod to disable or modify any component on the card.  In you case, it looks like you want to disable the network component.

Here the link in the Wiki to show how.
https://github.com/vasqued2/ha-teamtracker/wiki/Using-CARD-MOD-card-to-format-the-TEAMTRACKER-CARD

Let me know if you have any problems.

**Comment by romquenin** [comment]
Works prefectly, thanks @vasqued2

**Comment by KHV8** [comment]
How to determine the name of components to disable. The disable works perfectly, as shown in the documentation. However, Im not able to find a list of different components. Is there a list of a file where I can find them
